highcharts-angular/src/lib/highcharts-chart.directive.ts

```diff
@@ -84,7 +86,14 @@ export class HighchartsChartDirective {
   private keepChartUpToDate(): void {
     effect(() => {
       this.update();
-      this.chart()?.update(this.options(), true, this.oneToOne());
+      if (!this.chartCreated) {
+        const chart = this.chart();
+        if (chart) {
```

Comment:
Something is wrong with GH suggestions (cannot suggest changing more than 1 line, so probably need to check manually)

```suggestion
        if (this.chart()) {
```